### PR TITLE
YD-673 Mark fatal issues with ALERT

### PR DESCRIPTION
**Pull request by Bert-R**
This is done for internal server errors occurring in HTTP requests and for asynchronous processes (Quartz jobs, batch jobs and async tasks (CompletableFuture.runAsync).

Along with this:
* Renamed nu.yona.server.rest.Constants into RestConstants, to prevent clashes in places where both are needed
* The PASSWORD_HEADER constant was mostly statically imported. Did this now everywhere.

### CODE REVIEW

File: core/src/main/java/nu/yona/server/rest/RestConstants.java [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
- * Copyright (c) 2015, 2019 Stichting Yona Foundation This Source Code Form is subject to the terms of the Mozilla Public License,
+ * Copyright (c) 2015, 2020 Stichting Yona Foundation This Source Code Form is subject to the terms of the Mozilla Public License,
  * v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
  *******************************************************************************/
 package nu.yona.server.rest;
 
-public final class Constants
+public final class RestConstants
 {
 	public static final String PASSWORD_HEADER = "Yona-Password";
```
Comment (yonabuild) [inline core/src/main/java/nu/yona/server/rest/RestConstants.java]: BLOCKER 'PASSWORD' detected in this expression, review this potentially hard-coded credential. [rule](http://10.127.10.107:9000/coding_rules#rule_key=squid%3AS2068)

File: core/src/main/java/nu/yona/server/rest/RestConstants.java [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 	public static final String NEW_DEVICE_REQUEST_PASSWORD_HEADER = "Yona-NewDeviceRequestPassword";
```
Comment (yonabuild) [inline core/src/main/java/nu/yona/server/rest/RestConstants.java]: BLOCKER 'PASSWORD' detected in this expression, review this potentially hard-coded credential. [rule](http://10.127.10.107:9000/coding_rules#rule_key=squid%3AS2068)

**Comment by yonabuild** [comment]
SonarQube analysis reported 2 issues
* ![BLOCKER][BLOCKER] 2 blocker

Watch the comments in this conversation to review them.

[BLOCKER]: https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER'

File: core/src/main/java/nu/yona/server/rest/RestConstants.java (hunk 1 above)
Comment (Bert-R) [reply core/src/main/java/nu/yona/server/rest/RestConstants.java]: This obviously is a false positive.

File: core/src/main/java/nu/yona/server/rest/RestConstants.java (hunk 2 above)
Comment (Bert-R) [reply core/src/main/java/nu/yona/server/rest/RestConstants.java]: This obviously is a false positive.

**Pull request merged**